### PR TITLE
CC3XX: Add entrypoint for key agreement

### PR DESCRIPTION
**Pull request by noonfom**
Signed-off-by: Abbas Bracken Ziad <abbas.brackenziad@arm.com>

### CODE REVIEW

**Review by adeaarm — DISMISSED** [review]
LGTM

File: library/psa_crypto.c [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 #include "common.h"
 
-#if defined(MBEDTLS_PSA_CRYPTO_C)
+//#if defined(MBEDTLS_PSA_CRYPTO_C)
```
Comment (adeaarm) [inline library/psa_crypto.c:23]: why do we need this?

Comment (noonfom) [reply library/psa_crypto.c:23]: It's a mistake, will fix

**Review by adeaarm — APPROVED** [review]
LGTM

**Pull request merged**